src/vcf/validate_optional_policy.cpp

```diff
@@ -99,8 +99,8 @@ namespace ebi
                     if (alleles.size() != ploidy) {
                         throw new SamplesFieldBodyError{
                                 state.n_lines,
-                                "Sample #" + std::to_string(i) + " has " + std::to_string(alleles.size())
-                                        + " allele(s), but " + std::to_string(ploidy) + " were found in others",
+                                "Sample #" + std::to_string(i) + " has " + std::to_string(alleles.size()) + " allele(s)",
+                                "But " + std::to_string(ploidy) + " were found in others",
```

Comment:
Leaving a note to myself to review this case better

Reply:
The detailed message could be slightly modified, otherwise the punctuation is going to make it look quite weird. Something like

```
std::to_string(ploidy) + " alleles were found in other samples"
```